app.js

```diff
@@ -46,6 +46,10 @@ app.all('*', function (req, res, next) {
   }
 })
 
+app.get('/help/about', function (req, res, next) {
```

Comment:
We are not calling `next` handler here so there is no reason to pass it to handler here.